src/components/JUPTable/JUPTable.tsx

```diff
@@ -30,7 +30,7 @@ const TableTitle: React.FC<ITableTitleProps> = ({ title, path, DisplayedComponen
   return (
     <>
       {title && (
-        <TitleText variant="h6" id="tableTitle">
+        <TitleText key={`title-${title}`} variant="h6" id="tableTitle">
```

Comment:
traditionally we've done something to replace/remove special characters and re-format the key string a little. Eg, replace spaces with underscores.

This isn't a problem imo but is an easy change if you see any value in it.

Reply:
fixed